common/scripts/get_source.sh

```diff
@@ -194,6 +210,7 @@ if [ $BAND == "SR" ]; then
 else
     get_bsa_src
 fi
+
```

Comment:
On line 205, please add the below code:
if [ $TARGET_ARCH == "arm" ]; then
    sudo apt-get install libmpc-dev
fi

Resolution of Linux build issue seen in some envs

Reply:
+1. I'll append a fixup patch to the series. I'll squash the fixup commit at next rebase of this P-R.